src/models.py

```diff
@@ -333,15 +339,12 @@ def build_embeddings(args, vocab, tasks, pretrained_embs=None):
         )
         token_embedders["words"] = embeddings
         d_emb += d_word
-    else:
-        embeddings = None
-        log.info("\tNot using word embeddings!")
 
     # Handle cove
     cove_layer = None
     if args.cove:
         assert embeddings is not None
-        assert args.word_embs == "glove", "CoVe requires GloVe embeddings."
+        assert args.input_module == "glove", "CoVe requires GloVe embeddings."
```

Comment:
Does it make sense to fold cove into this logic as well? Not sure I know that part of the code well enough.

Reply:
I don't think so since cove is the only one that relies on another word embedding (glove). Plus, i'm also not the most familiar with it and am not as comfortable putting it in input_modules alongside the others atm.

Reply:
Mind opening an issue? Seems okay to leave for now, but it will make the documentation/code harder to follow. Ultimately, we should fold the glove loading logic into cove, and treat cove as the input module.